Affiliated-project-FAQ.md

```diff
@@ -11,6 +11,8 @@ our [affiliated project policy](Affiliated-Project-Policy.md), please
 - Proof of compliance with Affiliate Status requirements where possible, e.g.
 licence, code of conduct, etc. or plan to implement any missing requirements in the near future.
 
+Upon submitting this issue template, your project will become an OBF Candidate project and will be eligible to be considered for Affiliated project status, at an OBF public meeting, generally after one year of Candidate status. 
```

Comment:
I think we should not restate or paraphrase policy in the FAQ. Specifically, the process for attaining Candidate or Affiliate OBF Project is in the policy, and is not consistent with this sentence. (For example, simply submitting this form does not make a project a Candidate OBF Project, and it's not clear whether an _OBF public meeting_ is a BOSC meeting or BoD meeting; the latter is not required in the policy.)
```suggestion
Filling out and submitting this issue template constitutes a formal request to be put on the next ballot for voting on OBF affiliation status. If your project has not previously been affiliated with the OBF, your application is for attaining Candidate OBF Project status. If your project is currently a Candidate OBF Project, then your application is for attaining Affiliated OBF Project status. 
```

Reply:
Perhaps my version is still restating policy too much (such as which status comes first), but I wasn't sure how to say more succinctly which type of application one is submitting.

Reply:
I think I was seeing things slightly differently, and if you applied for candidate status it would automatically be accepted, since it doesn't mean anything much really (no benefits or obligations) except that we'll then _consider_ you for affiliate status. Moving manually through the two tiers seems fine too.